http.c

```diff
@@ -123,6 +123,8 @@ static http_curlopt settable_curlopts[] = {
 	{ "CURLOPT_TIMEOUT", CURLOPT_TIMEOUT, CURLOPT_LONG, false },
 	{ "CURLOPT_TIMEOUT_MS", CURLOPT_TIMEOUT_MS, CURLOPT_LONG, false },
 	{ "CURLOPT_CONNECTTIMEOUT", CURLOPT_CONNECTTIMEOUT, CURLOPT_LONG, false },
+	{ "CURLOPT_SSL_VERIFYHOST", CURLOPT_SSL_VERIFYHOST, CURLOPT_LONG, false },
+	{ "CURLOPT_SSL_VERIFYPEER", CURLOPT_SSL_VERIFYPEER, CURLOPT_LONG, false },
```

Comment:
These options look to be relatively [recent](http://blog.cleverelephant.ca/2017/06/curl-options.html) so they might need to be lower down in one of the version-protected blocks. Otherwise, if you confirm they actually *work* in practice (build, install, run your code) then I'm happy to merge.

Reply:
buil, install and run work as expected